.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,33 @@
+name: Maintenance
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * *" # Runs every day at 12 PM (noon) UTC, which is 4 AM PST
+jobs:
+  prune:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-22.04
+    permissions:
+      # contents: write # only for delete-branch option
+      issues: write
+      pull-requests: write
+    name: Issue and PR maintenance
+    steps:
+      - name: Prune stale issues
+        uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 270 # 9 months
+          days-before-close: 30 # 1 month
+          stale-issue-label: stale
+          exempt-issue-labels: bug,help wanted
+          exempt-pr-labels: external-dependency
+          stale-issue-message: |
```

Comment:
I think [stale-issue-message](https://github.com/actions/stale#stale-issue-message) is when the issue transitioned to `stale`, `[close-issue-message](https://github.com/actions/stale#close-issue-message) is the comment added then the issue is closed.  The text seems to be for `close-issue-message`